x/mongo/driver/operation.go

```diff
@@ -273,6 +273,7 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 	var original error
 	var retries int
 	retryable := op.retryable(desc.Server)
+
```

Comment:
Minor, but I suggest reverting this whitespace change unless this is from formatting.